test_reporting/report_data_storage.py

```diff
@@ -193,6 +203,13 @@ def upload_reachability_data(self, ping_output: List) -> None:
             result.update({"UTCTimestamp": ping_time})
         self._ingest_data(self.TESTBEDREACHABILITY_TABLE, ping_output)
 
+    def upload_swss_report_file(self, swss_file: str) -> None:
+        """Upload a report to the back-end data store.
+        Args:
+            file: json
```

Comment:
docstring needs to comply with the actual argument list.

Reply:
thx, I solved it